packages/headless/src/controllers/core/facets/_common/facet-option-definitions.ts

```diff
@@ -38,3 +38,19 @@ const facetSearchOptionDefinitions: SchemaDefinition<FacetSearchRequestOptions>
 export const facetSearch = new RecordValue({
   values: facetSearchOptionDefinitions,
 });
+
+export const allowedValues = new RecordValue({
+  options: {required: false},
+  values: {
+    type: new StringValue({
```

Comment:
What does `type: "simple"` mean? Why is it only available on the core facet?

Reply:
That's how the API function. It means that for now it only support the `simple` mode, where we match allowed values exactly.

They added that to leave them room to add more complex match mode, like regex or wildcards if we need it eventually.